docs/nuget-org/package-readme-on-nuget-org.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Package Readme on NuGet.org
+description: Detailed explanation of how Readme files on NuGet.org are rendered and what to do when you run into issues.
+author: chgill-MSFT
+ms.author: chgill
+ms.date: 02/23/2021
+ms.topic: conceptual
+ms.reviewer: anangaur
+---
+
+# Package Readme on NuGet.org
+
+[Include a Readme file in your NuGet package](https://docs.microsoft.com/nuget/reference/msbuild-targets#packagereadmefile) to make your package details richer and more informative for your users!
```

Comment:
nit: "Readme" looks a little strange having just the first word capitalized. Would just "readme" work? It's a bit awkward either way cause it's not a real work 😄.
https://en.wikipedia.org/wiki/README -- this one was README but that would be yelling

Reply:
Let's just say README? That seems most common.

Reply:
I'm fine with whatever as long as we are self consistent!